frontend/lib/src/AppNode.ts

```diff
@@ -404,20 +404,17 @@ export class AppRoot {
   private readonly root: BlockNode
 
   /**
-   * Create an empty AppRoot with an optional placeholder element.
+   * Create an empty AppRoot with an optional placeholder "skeleton" element.
```

Comment:
nit: The skeleton will always be used in the empty state, or? 

```suggestion
   * Create an empty AppRoot with a placeholder "skeleton" element.
```

And one aspect to double-check would be if this has any negative impact on the `clearAppState` here since it did not show the `Loading...` placeholder:

https://github.com/streamlit/streamlit/blob/060f4f8840ba0c16558a3a9f16b4fa47e88b00d7/frontend/app/src/App.tsx#L1110

Reply:
> nit: The skeleton will always be used in the empty state, or?

Good point. Fixed.

> And one aspect to double-check would be if this has any negative impact on the clearAppState here since it did not show the Loading... placeholder:

Yeah, I had to change that or the skeleton would only appear in the short period where no websocket messages had been exchanged between the server and client.

Looking at the code, this change seems like the correct thing to do. Though I was definitely a little worried 😆 . So my plan was to do the following:

1. Play around with different Streamlit apps to see if I find any issues. I tried things like stopping/starting the server, changing to a different script, switching pages in an MPA, clearing the app in Python, etc.

   STATUS: I did all of the above and the app behaved as expected. 

2. Ask for Kasim to QA this specific aspect of the PR thoroughly.

Reply:
Sounds good 👍